samples/mpu/mem_domain_apis_test/README.rst

```diff
@@ -0,0 +1,96 @@
+.. _mem_domain_apis_test:
+
+Memory Domain APIs Test
+#######################
+
+Overview
+********
+
+This is a simple application that demonstrates the usage of Memory Domain APIs.
+Two memory domains (app0_domain and app1_domain) and three application threads
+(app thread 0, 1, 2) are created by the main thread.
+Each application thread will add itself into a memory domain while it starts.
+These threads will yield their execution alternately and we can observe the
+changes of domain partition regions during context switch via the serial console.
+
+Building and Running
+********************
+
+This project can be built and executed as follows:
+
+.. code-block:: console
+
+   $ cd samples/mpu/mem_domain
+   $ make BOARD=stm32f4_disco
+
+Connect the board to your host computer using the USB port.
+Flash the generated zephyr.bin on the board.
+Reset the board and you should be able to see on the corresponding
+Serial Port the following message:
+
+.. code-block:: console
+
+   ***** BOOTING ZEPHYR OS v1.8.99 - BUILD: Jul 11 2017 05:59:33 *****
+   initialize memory domains
+   [general] [DBG] configure_mpu_mem_domain: configure thread 0x20000480's domain
+   [general] [DBG] arm_core_mpu_configure_mem_domain: disable domain partition regions
+   [general] [DBG] arm_core_mpu_configure_mem_domain: disable region 0x6
+   [general] [DBG] arm_core_mpu_configure_mem_domain: disable region 0x7
+   app thread 0 (0x20000480) starts.
+   app thread 0 (0x20000480) adds itself into app0_domain.
+   [general] [DBG] configure_mpu_mem_domain: configure thread 0x20000480's domain
+   [general] [DBG] arm_core_mpu_configure_mem_domain: configure domain: app0_domain
+   [general] [DBG] arm_core_mpu_configure_mem_domain: set region 0x6 0x20000600 0x20
+   [general] [DBG] arm_core_mpu_configure_mem_domain: set region 0x7 0x200005e0 0x20
+   app thread 0 (0x20000480) yields.
+
+Sample Output
```

Comment:
is there some reason to separate the output shown above, with the output shown below?